src/projectscene/view/clipsview/clipslistmodel.h

```diff
@@ -127,5 +125,8 @@ private slots:
     //! Offset between mouse click position on clip's header and clip's start and end time
     double m_clipEditStartTimeOffset = -1.0;
     double m_clipEditEndTimeOffset = -1.0;
+
+    bool m_receiveAutoScrollNotifications = false;
+    QMetaObject::Connection m_autoScrollConnection;
```

Comment:
can we remove `m_receiveAutoScrollNotifications` and replace it with connection check?
```
if (m_autoScrollConnection) {
...
```